components/script/dom/webidls/HTMLTableCellElement.webidl

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // https://html.spec.whatwg.org/multipage/#htmltablecellelement
-[Abstract, Exposed=(Window,Worker)]
+[Abstract]
```

Comment:
Note: This `Abstract` is not exposed in the specifications.  I left it because it was already there before #12353 was done (#12415 is supposed to fix #12353).